config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   devise_for :users
   
+  resources :products
```

Comment:
you need to specify which routes that should be available

Reply:
Changes made accordingly.